### PR TITLE
Adicionar Isabel costa ao README

### PR DESCRIPTION
**Pull request by isabelcosta**
Adicionar Isabel e a minha cidade ao README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,5 @@
 Contains all of the students' names
 
 ### Contributors
-Inês Martins
\ No newline at end of file
+Inês Martins
+Isabel Costa Elvas
```
Comment (inesmartins) [inline README.md:6]: Please remove city, we only need the names, ty!

**Review by inesmartins — CHANGES_REQUESTED** [review]
Looks good, but left a couple of comments, please check them out, good job.

File: README.md (hunk 1 above)
Comment (isabelcosta) [reply README.md:6]: will do! Thank you @inesmartins

**Review by inesmartins — APPROVED** [review]
Looks good!

**Pull request merged**